thoth/storages/data/alembic/versions/76413cbe99bf_added_source_type_to_adviser_run_table.py

```diff
@@ -0,0 +1,72 @@
+"""Added source type to adviser_run table
+
+Revision ID: 76413cbe99bf
+Revises: 7c261487d4d4
+Create Date: 2020-06-29 17:13:45.349792+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '76413cbe99bf'
+down_revision = '7c261487d4d4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('adviser_run', sa.Column('source_type', postgresql.ENUM('CLI', 'KEBECHET', 'S2I', 'GITHUB_APP', 'JUPYTER_NOTEBOOK', name='source_type'), nullable=True))
+    op.create_foreign_key(None, 'adviser_run', 'external_software_environment', ['external_run_software_environment_id'], ['id'], ondelete='CASCADE')
```

Comment:
@fridex seems like its an alembic issue - https://github.com/sqlalchemy/alembic/issues/363#issuecomment-614962398
Should I just manually remove these?

Reply:
> Should I just manually remove these?

Yes, please do so.

Reply:
I tried removing the unnecessary migration statments, still seems to failing..
Any suggestion on this - https://github.com/sqlalchemy/alembic/issues/363#issuecomment-614962398
Should we have just write manual SQL statements with op.execute for the migration? Or is there a better alternative?
Complete error log - https://bit.ly/3dOIXux

Reply:
> Should we have just write manual SQL statements with op.execute for the migration? Or is there a better alternative?

Check migrations we already have. I don't recall if we did something similar to this (to learn from). Othrewise, raw  SQL statements will be necessary I guess.

Reply:
@fridex Thanks, I have added a manual migration, PTAL. :smile: